compiler/dyno/lib/uast/Reduce.cpp

```diff
@@ -41,6 +41,22 @@ owned<Reduce> Reduce::build(Builder* builder,
   return toOwned(ret);
 }
 
+owned<Reduce> Reduce::build(Builder* builder,
+                            Location loc,
```

Comment:
I think we probably only need the one builder, no? Ok to just add the argument to the previous builder.

Reply:
removed